cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,20 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | Postgre | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling & Repair| Yes | No | No |  |  | No | Yes 
+Synchronous Replication | Yes | No | No |  |  | No |  
```

Comment:
OK, I'll move Strong Consistency up here. Is it necessary to get the term "replication" in somewhere, though?